impress-org/givewp
# Refactor: update plugin banner with header image details.

Resolves #

## Description

This PR updates the plugin banner with a new video, title and description to reflect details for the new header image. A conditional check has been added to validate the  `givewp_welcome_banner_dismiss` options timestamp against a new start date. It should show if `givewp_welcome_banner_dismiss` is not in the database or if it has a unix timestamp that is before the start date.

The video playback has also been updated. Clicking on the video container now controls the playback for the video as opposed to just the button located on the bottom left.

## Affects

Plugins page + Welcome Banner

## Visuals


https://github.com/impress-org/givewp/assets/75056371/ef0dd6e0-90a1-43b8-8c31-efeb2c37f318




## Testing Instructions
- Visit the plugin page the banner should be displaying. 
- Verify the banner video is playable.
- Close the banner -> refresh the page. Verify that no longer is visible & remains hidden.

## Pre-review Checklist
